rocketpy/EnvironmentAnalysis.py

```diff
@@ -382,10 +399,18 @@ def __localize_input_dates(self):
     def __find_preferred_timezone(self):
         if self.preferred_timezone is None:
             # Use local timezone based on lat lon pair
-            tf = TimezoneFinder()
-            self.preferred_timezone = pytz.timezone(
-                tf.timezone_at(lng=self.longitude, lat=self.latitude)
-            )
+            try:
+                timezonefinder = import_optional_dependency("timezonefinder")
+                tf = timezonefinder.TimezoneFinder()
+                self.preferred_timezone = pytz.timezone(
+                    tf.timezone_at(lng=self.longitude, lat=self.latitude)
+                )
+            except ImportError:
+                print(
+                    "'timezonefinder' not installed, defaulting to UTC."
+                    + " Install timezonefinder to get local time zone."
+                )
```

Comment:
Perhaps, to make the error messages more consistent, suggest the command to install `timezonefinder`.

Reply:
Solved: [7cf2435](https://github.com/RocketPy-Team/RocketPy/pull/403/commits/7cf2435ed069f331608bafd471081888d3e40ad4)